mdoc/src/main/scala/mdoc/internal/markdown/MarkdownFile.scala

```diff
@@ -41,12 +41,14 @@ object MarkdownFile {
         backtickStart: Int,
         backtickEnd: Int
     ): CodeFence = {
-      val open = newText(state.start, state.start + state.backticks.length())
+      val tag = newText(state.start, state.start + state.indent)
```

Comment:
Could you maybe describe what are the tags? Maybe we should put it behind an additional setting as suggested by @olafurpg ?

Reply:
in here, tag is what precedes the opening code fence
```
abc ```
    foo
    bar
    ```
``` 

in this example `abc ` is a tag

Reply:
maybe `tag` is not the best name. if you can think off better name, please go ahead